elao.app/README.md

```diff
@@ -554,7 +563,7 @@ secrets@staging:
 	gomplate --input-dir=secrets/staging --output-map='{{ .in | replaceAll ".gohtml" "" }}'
 ```
 
-Put your templates in `.gohtml` files inside a `secrets/[production|staging]` directory at the root of the project.  
+Put your templates in `.gohtml` files inside a `secrets/[production|staging]` directory at the root of the project.
```

Comment:
```suggestion
Put your templates in `.gohtml` files inside a `secrets/[production|staging]` directory at the root of the project.  
```

(two spaces at the end of a line means a newline in markdown, this is intended and should be reverted)

Reply:
I've learned something today :)
Wouldn't it be easier to add real newlines ? I guess this spaces removal is due to the code editor itself